Homebrew/Feats/Book of the Righteous.xml

```diff
@@ -26,7 +26,6 @@
 		<text>The Breath speaks for the Unity, learning to understand and influence people as a leader and lawmaker, and to comprehend the world for her sisters, just as Naryne did on the Journey to the East. You can only select this feat at the same time as other characters take the feats for The Arm and The Eye, and you can never take either of those feats. You and those characters form your Unity. You gain the following benefits:</text>
 		<text>• Increase your Wisdom or Charisma by 1, to a maximum of 20.</text>
 		<text>• You can communicate with the members of your Unity telepathically, even if you are not on the same plane.</text>
-		<text>• When you cast a spell with a range of touch,another member of your Unity can deliver the spell as if they had cast the spell. That member of the Unity must be within 100 feet of you, and they must use their reaction to deliver the spell when you cast it. If the spell requires an attack roll, you use your attack modifier for the roll.</text>
```

Comment:
Not sure why this was deleted, I think you meant to add a space after the comma

Reply:
good catch 👍 fixed